src/test/java/com/ocient/cli/extract/writer/CustomTsvWriterTest.java

```diff
@@ -151,4 +151,26 @@ public void testEnclosureRequired() {
         assertEquals("\"a\"|\"b\"|\"c|d\"|\"ef\"\"crazy\"\"\"\n",
                 byteArrayOutputStream.toString(StandardCharsets.UTF_8));
     }
+
+    /**
+     * Escape: \\\\
```

Comment:
I think this may be a bit confusing. I'd recommend putting the _real_ char here:

```suggestion
     * Escape: \
```